test/cluster.sh

```diff
@@ -50,6 +50,7 @@ function install_knative_serving() {
   echo ">> Bringing up Serving"
   # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
   kubectl apply -f "${RELEASE_NO_MON_YAML}" || return 1
```

Comment:
Looks like there's a merge mix up here. This line shouldn't be here, otherwise serving is being installed twice. The comment should be moved to line 55.

Reply:
Yeah. I was wondering about that as well :) Updated.